lib/shelley/test/integration/Main.hs

```diff
@@ -400,7 +400,7 @@ withTracers testDir action = do
         ekgEnabled >>= flip when (EKG.plugin cfg walTr sb >>= loadPlugin sb)
         withLogging testLogOutputs $ \(_, (_, testTr)) -> do
             let trTests = appendName "integration" testTr
-            let tracers = setupTracers (tracerSeverities (Just Info)) walTr
+            let tracers = setupTracers (tracerSeverities (Just Debug)) walTr
```

Comment:
This won't change any default behaviour in an unintended way, right?

Reply:
Nope - not a problem for tests. There is a global minimum severity, controlled by environment variable, with default Critical (stdout scribe) and Info (`wallet.log` scribe). The minimum severity for any given tracer is the minimum of the global minSev and the tracer minSev.